formula/src/main/java/com/instacart/formula/Inspector.kt

```diff
@@ -88,8 +88,8 @@ interface Inspector {
      *
      * @param formulaType Formula type used to filter for specific events.
      * @param result Transition result used to check what kind of transition.
-     * @param requiresEvaluation Indicates if transition requires a new evaluation.
+     * @param evaluate Indicates if transition requires a new evaluation.
```

Comment:
Simpler name

Reply:
I guess we can rename without backwards compatibility since no releases were done between this class getting created and it releasing as a version you can depend on?

Reply:
Yep, and also we don't provide such strong backwards compatibility guarantees (as it's pre-1.0).